Data/future/cbot/margins/BCF.csv

```diff
@@ -1,17 +1,10 @@
 date,initial,maintenance
-# https://www.cmegroup.com/notices/clearing/2017/12/Chadv17-490.html
 20171215,523,475
-# https://www.cmegroup.com/notices/clearing/2018/06/Chadv18-223.html
+20171218,523,475
```

Comment:
Duplicated. Seems it might be related to the fact that 475 * 1.1 is a decimal value?